### PR TITLE
cuDNN v8.2.0

### PR DESCRIPTION
**Pull request by takagi**

### CODE REVIEW

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Comment by kmaehashi** [comment]
@takagi Could you enable cuDNN for `cupy-py36 ` in `run_test.py`?

**Comment by takagi** [comment]
Also `cupy-slow`?

**Comment by kmaehashi** [comment]
I think we can keep cuDNN 8.1 in `cupy-slow` for test coverage with various cuDNN versions.

pfnCI, test this please.

**Comment by takagi** [comment]
Okay. Makes sense.

**Comment by takagi** [comment]
I will also post another PR to remove py35 related things.

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Comment by kmaehashi** [comment]
@takagi Combination test variant seems using Python 3.5. Could you take a look on https://github.com/chainer/chainer-test/pull/654 ?

**Comment by takagi** [comment]
pfnCI, test this please.

**Comment by takagi** [comment]
pfnCI, test this please.

File: docker.py [hunk 1]
```diff
@@ -947,17 +947,17 @@ def make_dockerfile(conf):
     dockerfile += codes[conf['nccl']]
     dockerfile += codes[conf['cutensor']]
 
+    # Update pip to v20 which is the last version supporting Python 3.5.
+    dockerfile += '''
+RUN pip install -U 'pip<21' && rm -rf ~/.cache/pip
```
Comment (emcastillo) [inline docker.py:952]: Lets also do
`RUN pip install "six>=1.9"` here

**Comment by emcastillo** [comment]
Jenkins, test this please

**Comment by takagi** [comment]
Jenkins, test this please

**Comment by takagi** [comment]
Rebased on https://github.com/chainer/chainer-test/pull/654 and cleaned up the commits.

**Comment by takagi** [comment]
Jenkins, test this please

**Pull request merged**